src/OrchardCore/OrchardCore.Recipes.Abstractions/Services/IRecipeReader.cs

```diff
@@ -6,6 +7,10 @@ namespace OrchardCore.Recipes.Services
 {
     public interface IRecipeReader
     {
-        Task<RecipeDescriptor> GetRecipeDescriptor(string recipeBasePath, IFileInfo recipeFileInfo, IFileProvider fileProvider);
+        [Obsolete("This method has been deprecated, please use GetRecipeDescriptorAsync(string, IFileInfo, IFileProvider).")]
+        async Task<RecipeDescriptor> GetRecipeDescriptor(string recipeBasePath, IFileInfo recipeFileInfo, IFileProvider fileProvider)
+            => await GetRecipeDescriptorAsync(recipeBasePath, recipeFileInfo, fileProvider);
```

Comment:
Here you can remove the `async` and `await` keywords.

Reply:
Yes, we could